build.zig

```diff
@@ -11,12 +11,13 @@ pub fn build(b: *std.Build) void {
         .optimize = optimize,
     });
 
-    lib.addIncludePath(b.path("include"));
     lib.addCSourceFiles(.{ .files = &generic_src_files });
     lib.defineCMacro("SDL_USE_BUILTIN_OPENGL_DEFINITIONS", "1");
     lib.linkLibC();
+    var use_pregenerated_config = false;
```

Comment:
`const use_pregenerated_config = switch (t.os.tag) { ...`

as a rule of thumb, `var` should be used pretty much only in loops.